test/metabase/test/util.clj

```diff
@@ -788,7 +791,9 @@
                        additional-conditions (with-model-cleanup-additional-conditions model)]]
           (t2/query-one
            {:delete-from (t2/table-name model)
-            :where       [:and max-id-condition additional-conditions]}))))))
+            :where       [:and max-id-condition additional-conditions]})
+          ;; TODO we don't (currently) have index update hooks on deletes, so we need this to ensure rollback happens.
```

Comment:
Thinking of reviving the delete code from the dead, so we can call it manually in a few places, even though the hook is infeasible.

Reply:
Hmmm I think that actually makes sense.